File: apps/Android/MnnLlmChat/app/src/main/java/com/alibaba/mls/api/ms/MsRepoInfo.java
// Created by ruoyi.sjd on 2025/2/10.
// Copyright (c) 2024 Alibaba Group Holding Limited All rights reserved.

package com.alibaba.mls.api.ms;

import java.util.List;


public class MsRepoInfo {
    public int Code;
    public Data Data;
    public String Message;
    public boolean Success;

    public static class Data {
        public List<FileInfo> Files;
    }

    public static class FileInfo {
        public String Name;
        public String Path;

        public String Revision;
        public long Size;

        public String Sha256;
    }
}

File: apps/Android/MnnLlmChat/app/src/main/java/com/alibaba/mls/api/ms/MsRepoInfo.kt
// Created by ruoyi.sjd on 2025/2/10.
// Copyright (c) 2024 Alibaba Group Holding Limited All rights reserved.
package com.alibaba.mls.api.ms

class MsRepoInfo {
    var Code: Int = 0
    var Data: ResponseData? = null
    var Message: String? = null
    var Success: Boolean = false

    class ResponseData {
        var Files: List<FileInfo>? = null
        var LatestCommitter:Committer? = null
    }

    class Committer {
        var CreatedAt:Long? = null
    }

    class FileInfo {
        var Name: String? = null
        var Path: String? = null

        var Revision: String? = null
        var Size: Long = 0

        var Sha256: String? = null
        var Type: String? = null
    }
}